src/app/singletons/fighter_manager.rs

```diff
@@ -143,16 +143,16 @@ mod impl_ {
         pub(super) fn set_visible_finalbg(manager: *mut app::FighterManager, visible: bool);
     
         #[link_name = "_ZN3app8lua_bind33FighterManager__show_finalbg_implEPNS_14FighterManagerEj"]
-        pub(super) fn show_finalbg(manager: *mut app::FighterManager, arg: u32);
+        pub(super) fn show_finalbg(manager: *mut app::FighterManager, arg2: u32);
     
         #[link_name = "_ZN3app8lua_bind34FighterManager__start_finalbg_implEPNS_14FighterManagerEif"]
-        pub(super) fn start_finalbg(manager: *mut app::FighterManager, arg: i32, arg2: f32);
+        pub(super) fn start_finalbg(manager: *mut app::FighterManager, arg2: i32, arg3: f32);
     
         #[link_name = "_ZN3app8lua_bind32FighterManager__start_movie_implEPNS_14FighterManagerEf"]
-        pub(super) fn start_movie(manager: *mut app::FighterManager, arg: f32);
+        pub(super) fn start_movie(manager: *mut app::FighterManager, arg2: f32);
     
         #[link_name = "_ZN3app8lua_bind48FighterManager__start_movie_on_rendering_2d_implEPNS_14FighterManagerEf"]
-        pub(super) fn start_movie_on_rendering_2d(manager: *mut app::FighterManager, arg: f32);
+        pub(super) fn start_movie_on_rendering_2d(manager: *mut app::FighterManager, arg2: f32);
```

Comment:
I won't fight you on these but I did originally decide to do it the other way. I don't remember why so I don't really care one way or the other.

Reply:
Feel free to revert them in your own commits if it's not an issue with this pull request. I personally found it to be clearer to know the placement of the unknown parameter, and I think I changed it in the first place because it wasn't consistent with other modules in the crate.